lib/autoyast.pm

```diff
@@ -800,6 +800,12 @@ sub generate_json_profile {
 
     my @profile_options = map { " --tla-" . (/true|false/ ? "code" : "str") . " $_" }
       split(' ', trim(get_var('AGAMA_PROFILE_OPTIONS')));
+
+    my @extra_vars = qw(PACKAGES PATTERNS);
+    for my $var (@extra_vars) {
+        next unless my $value = get_var($var);
+        push @profile_options, "--ext-str " . $var . "=" . $value;
+    }
```

Comment:
I would postpone this tbh, anyway we can add in job group configuration %PATTERNS% so it is more clear, there is no internal magical additiions.

Reply:
OK.